taichi/common/serialization.h

```diff
@@ -203,7 +203,7 @@ class Serializer {
   };
 };
 
-inline std::vector<uint8> read_data_from_file(const std::string &fn) {
```

Comment:
I think this was a typo in the original code, because `uint8_t` is used everywhere else. This will start to cause compile errors once we `enable_language(CUDA)` in cmake.